modules/api/src/main/java/org/apache/ignite/schema/builder/SchemaTableBuilder.java

```diff
@@ -35,6 +35,14 @@
      */
     SchemaTableBuilder columns(Column... columns);
 
+    /**
+     * Adds columns to the table.
+     *
+     * @param columns Table columns.
+     * @return {@code This} for chaining.
+     */
+    SchemaTableBuilder columns(Iterable<Column> columns);
```

Comment:
Let's drop `SchemaTableBuilder columns(Column... columns) `method,
as user always can call builder.columns(List.of(Columns[])